pages/participate/participate.md

```diff
@@ -25,8 +25,20 @@ include, but are not limited to:
 - Training and workforce development
 - Maturing and expanding the RSE profession
 
+**Submission Website**: [EasyChair](https://easychair.org/conferences/?conf=usrse2024);
+please see submission templates below for each type of submission.
+
+**Review and Publication**: All contributions will be subject to standard peer
+review for quality, relevance, and novelty, which will determine acceptance and the form of
```

Comment:
the form of presentation isn't part of the review process - form is what was submitted.  The reviewers only say yes or no to if it is presented in that format.